package.json

```diff
@@ -247,6 +252,12 @@
         "category": "ONE"
       }
     ],
+    "keybindings": [
+      {
+          "command": "one.explorer.runSingleSelectedCfg",
+          "key": "ctrl+f5"
```

Comment:
What about using `ctrl+f7`? Because this operation compiles the cfg file to generate the tvn binary. In Visual studio, `ctrl+f7` uses as the meaning of compilation. It would be more appropriate to use this key binding to run tvn or simulator or target.